### ISSUE TITLE
Translate the readme to different languages?

### ISSUE THREAD

**Issue by Andry60351** [razeware/emitron-iOS#505]
**Translating the readme to other languages**

Hi there! I was thinking that it would be a great idea if we could translate the readme into different languages!

I can start with translating the readme in Italian, as I am a native Italian speaker who is fluent in the English language as well!

I think this could really benefit the community and help massively people who don't speak/understand English very well!

**Comment by 0xTim** [comment]
@Andry60351 yep this would definitely be welcomed! If you want to, you can submit a PR with a `README.it.md` and add a link to it in the main README that would be awesome!

**Comment by Andry60351** [comment]
Sounds good! I'll work on it this week end 👍

**Comment by SuzGupta** [comment]
I think we could do this (by which I mean open source the translation effort) for free via [CrowdIn](https://crowdin.com/page/open-source-project-setup-request) and we would "just" need to integrate the resulting translations. [Open Food Facts](https://translate.openfoodfacts.org/) is one example of a project doing this.
